provider/cmd/pulumi-resource-azure-static-website/website.ts

```diff
@@ -134,36 +165,71 @@ export class Website extends pulumi.ComponentResource {
 
             this.cdnURL = pulumi.interpolate`https://${endpoint.hostName}`;
 
-            if (false) {
+            if (this.withCustomDomain) {
```

Comment:
Can this be inferred from the existence of `domain` (or whatever we call it)?

Reply:
true, we can